PokeAlarm/Events/MonEvent.py

```diff
@@ -132,6 +135,11 @@ def generate_dts(self, locale, timezone, units):
             'weather': weather_name,
             'weather_or_empty': Unknown.or_empty(weather_name),
             'weather_emoji': get_weather_emoji(self.weather_id),
+            'boosted_weather_id': self.boosted_weather_id,
+            'boosted_weather': boosted_weather_name,
+            'boosted_weather_or_empty': Unknown.or_empty(boosted_weather_name),
+            'boosted_weather_emoji':
```

Comment:
`<boosted_weather_emoji>` does not resolve. 

![2018-01-22 13_58_03- rare-pokemon-spawns - discord](https://user-images.githubusercontent.com/3013565/35238712-543c0da2-ff7c-11e7-9271-1e6329c4e211.png)